tests/acceptance/standard/test_group_avatar.py

```diff
@@ -0,0 +1,292 @@
+              avatar: "{self.test_image_path}"
+        """
+        run_gitlabform(config, group)
+
+        # Refresh group data
+        group = group.manager.get(group.id)
+
+        # Verify avatar is set
+        assert group.avatar_url is not None
+
+    def test__group_avatar_paths_comprehensive(self, group):
+        """Test both absolute and relative paths comprehensively"""
+
+        # Test 1: Absolute path (already tested above, but included for completeness)
+        config_absolute = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_settings:
+              avatar: "{self.test_image_path}"
+        """
+        run_gitlabform(config_absolute, group)
+
+        group = group.manager.get(group.id)
+        assert group.avatar_url is not None
+
+        # Test 2: Simple relative path
+        original_cwd = os.getcwd()
+        temp_dir = tempfile.mkdtemp()
+
+        try:
+            os.chdir(temp_dir)
+
+            # Copy image to temp directory
+            temp_image_name = "relative_test_avatar.png"
+            shutil.copy2(self.test_image_path, temp_image_name)
+
+            config_relative = f"""
+            projects_and_groups:
+              {group.full_path}/*:
+                group_settings:
+                  avatar: "{temp_image_name}"
+            """
+            run_gitlabform(config_relative, group)
+
+            group = group.manager.get(group.id)
+            assert group.avatar_url is not None
+
+        finally:
+            os.chdir(original_cwd)
+            shutil.rmtree(temp_dir)
+
+        # Test 3: Relative path with subdirectory
+        temp_dir2 = tempfile.mkdtemp()
+        try:
+            os.chdir(temp_dir2)
+
+            subdir = "test_images"
+            os.makedirs(subdir)
+            relative_subdir_path = os.path.join(subdir, "subdir_avatar.png")
+            shutil.copy2(self.test_image_path, relative_subdir_path)
+
+            config_relative_subdir = f"""
+            projects_and_groups:
+              {group.full_path}/*:
+                group_settings:
+                  avatar: "{relative_subdir_path}"
+            """
+            run_gitlabform(config_relative_subdir, group)
+
+            group = group.manager.get(group.id)
+            assert group.avatar_url is not None
+
+        finally:
+            os.chdir(original_cwd)
+            shutil.rmtree(temp_dir2)
+
+    def test__group_avatar_delete(self, group):
+        # First ensure there's an avatar to delete by setting one
+        config_set = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_settings:
+              avatar: "{self.test_image_path}"
+        """
+        run_gitlabform(config_set, group)
+
+        # Refresh group data to get current state
+        group = group.manager.get(group.id)
+
+        # Verify that avatar exists
+        assert group.avatar_url is not None, "Avatar should exist after setting it"
```

Comment:
I think we'd expect the group avatar to be set by the previous test, right? Do really need to run through gitlabform again before we begin the test? Wouldn't the last assert be sufficient before the test begins?